internal/file/file_manager_service.go

```diff
@@ -37,7 +37,13 @@ import (
 //go:generate go run github.com/maxbrunsfeld/counterfeiter/v6@v6.8.1 -generate
 //counterfeiter:generate . fileManagerServiceInterface
 
-const maxAttempts = 5
+const (
+	maxAttempts     = 5
+	addAction       = "add"
+	updateAction    = "update"
+	deleteAction    = "delete"
+	unchangedAction = "unchanged"
```

Comment:
Would it be better to use an enum here and have it defined in the `internal/model/file.go`?